packages/hooks/src/usePlatformRealAccounts.ts

```diff
@@ -0,0 +1,29 @@
+import { useStore } from '@deriv/stores';
+
+/**
+ * we can use this hook to get all real accounts for both Eu and Non-Eu regions.
+ * it loops through the accounts list and returns all real accounts
+ * @example const real_accounts = usePlatformRealAccounts();
+ * @returns [{ balance: 100, currency: 'USD' }, { balance: 50, currency: 'EUR' }]
+ */
```

Comment:
It's nice to have documentation like this in the code, But it will get outdated and invalid in less than a month and there is no way to make sure it's updated and correct or not.
Already the return type in the documentation is not correct as it differs from what TS says, In my opinion, we should rely on TS and good variable names for documenting the code, And we can document the business logic in MD files.

<img width="668" alt="image" src="https://user-images.githubusercontent.com/72082844/231036503-f82f4c48-b394-4209-aa46-2ce3f85e73a5.png">

cc: @balakrishna-deriv @yashim-deriv @mohsen-deriv @ali-hosseini-deriv @matin-deriv What do you guys think? 🤔

Reply:
yep jsdoc is da bomb

Reply:
so just keep the description and remove the return and example?